token-minter/sources/modules/signed_transaction.move

```diff
@@ -0,0 +1,97 @@
+module minter::signed_transaction {
```

Comment:
nit: add a comment section above this comment to explain what this module is doing about proof challenge and proof data validation? and then prob share couple potential use cases on how signed txn can be utilized.

Reply:
Done!